docs/docs/getting-started/bare-metal.md

```diff
@@ -0,0 +1,18 @@
+# Prepare a Bare Metal Instance
+
+## Hardware & Firmware Setup
+
+1. Update your BIOS to a version that supports AMD SEV-SNP. Updating to the latest available version is recommended as newer versions will likely contain security patches for AMD SEV-SNP.
```

Comment:
Is there a way for a user to check it is supported by the current bios version?

Reply:
Not in general no, they'd have to ask the motherboard vendor. Still, though, they really should upgrade.

Upgrading the BIOS of a server-class motherboard shouldn't be that much of a risk because most of them allow restoring the previous version after an upgrade using the BMC. It's much less risky than upgrading the BIOS on a consumer-class motherboard.